tensorflow_io/bigquery.md

```diff
@@ -95,7 +103,28 @@ if __name__ == '__main__':
 
 ```
 
-It also supports reading BigQuery column with repeated mode (each field contains array of values with primitive type: Integer, Float, Boolean, String, but RECORD is not supported). In this case, selected_fields needs be a dictionary in a
+In some cases when when row is too wide (has 30+ columns) or if you are doing batching with large batch sizes it might be benefitial to do batching before interleave to get a better performance. Here is an exaple showing how to do that:
```

Comment:
duplicate `when` in first line.

Reply:
`benefitial` --> `beneficial` in second line

Reply:
`exaple` --> `example` in third line

Reply:
so many typos, thanks for reviewing